.requirements

```diff
@@ -6,11 +6,13 @@ OPENSSL=3.2.1
 PCRE=10.43
 LIBEXPAT=2.5.0
 
+# Note: git repositories can be loaded from local path if path is set as value
+
 LUA_KONG_NGINX_MODULE=691ba795ced07364d491e8abbdf0c8c8d3778c14 # 0.10.0
 LUA_RESTY_LMDB=7d2581cbe30cde18a8482d820c227ca0845c0ded # 1.4.2
 LUA_RESTY_EVENTS=8448a92cec36ac04ea522e78f6496ba03c9b1fd8 # 0.2.0
 LUA_RESTY_WEBSOCKET=60eafc3d7153bceb16e6327074e0afc3d94b1316 # 0.4.0
-ATC_ROUTER=1eeb0509a90494dc8618c5cd034ca4be231bb344 # 1.6.1
+ATC_ROUTER=4a77f5a321ba56040f65626ea626f84a3ee6e991
```

Comment:
Seems no such release. Is it just a test purpose?

Reply:
yes, this is in a revert me commit.